projects/plugins/jetpack/tools/webpack.config.css.js

```diff
@@ -164,9 +164,9 @@ for ( const name of [
 	'modules/shortcodes/css/recipes',
 	'modules/shortcodes/css/recipes-print',
 	'modules/shortcodes/css/slideshow-shortcode',
-	'modules/contact-form/css/editor-inline-editing-style',
-	'modules/contact-form/css/editor-style',
-	'modules/contact-form/css/editor-ui',
+	'jetpack_vendor/automattic/jetpack-forms/src/contact-form/css/editor-inline-editing-style',
```

Comment:
Couldn't we remove those altogether, since they are built in the package via the `projects/packages/forms/tools/webpack.config.contact-form.js` config?

Reply:
Yes definitely :) I just realized the issue now on rebuilding locally, and spotting some untracked rtl CSS files being generated.